api/src/GTAVersions/Controllers/AuthController.cs

```diff
@@ -31,5 +34,16 @@ public async Task<IActionResult> Signup([FromBody] SignUpUserDTO model)
 
             return Ok(token);
         }
+
+        [Authorize]
+        [HttpPut("settings")]
```

Comment:
Rename this changePassword and change method to PATCH (Read difference between PUT and PATCH)